Gems/Atom/RHI/Code/Source/RHI/RayTracingPipelineState.cpp

```diff
@@ -69,6 +69,13 @@ namespace AZ
             return this;
         }
 
+        RayTracingPipelineStateDescriptor* RayTracingPipelineStateDescriptor::CallableShaderName(const AZ::Name& callableShaderName)
+        {
+            AZ_Assert(m_shaderLibraryBuildContext && m_hitGroupBuildContext == nullptr, "CallableShaderName can only be added to a ShaderLibrary");
```

Comment:
Do we need to check for m_hitGroupBuildContext == nullptr here?

Reply:
Yes, since it's invalid to put CallableShader in a HitGroup.  RayGen and Miss perform the same check.